README.md

```diff
@@ -1,14 +1,30 @@
 # OpenStack K8S Operators Architectures
 
-This repository may be used with to create validated
-architectures ([VAs](examples/va)), represented as CRs for
-[openstack-k8s-operators](https://github.com/openstack-k8s-operators).
-It may also be used to create deployed topologies
-([DTs](examples/dt)) which should only be used for testing.
+This repository contains templates used are part of the validated architecture
```

Comment:
```suggestion
This repository contains templates used as a part of the validated architecture
```

Reply:
@fultonj so this is how I normally handle suggestions in case you're using the UI. I'm not sure how picky we are about flat commit logs. In STF we prefer verbosity and history over commit cleanliness, but that might not be the expectation here. My default is using the suggestion method (which provides an extra commit) and then squashing commits at the end, if there isn't already a strong preference.